### ISSUE TITLE
Data Leakage in pre-processing

### ISSUE THREAD

**Issue by parthkl021** [apache/fineract-credit-scorecard#20]
In reference to this snippet from the preprocessing section in the file [model_notebook.ipynb](https://github.com/apache/fineract-credit-scorecard/blob/develop/ml/model_notebook.ipynb)
```
for col in data.columns:
        if(col not in categorical):
            data[col] = (data[col].astype('float') - np.mean(data[col].astype('float')))/np.std(data[col].astype('float'))
```
The normalization is done on the whole data (i.e. when the train and test split did not occur). This means that information from the test set is used to scale the training set.
